voluncheer/templates/volunteer/vol_org_view.html

```diff
@@ -58,10 +55,30 @@ <h2>POSTED OPPORTUNITIES</h2>
             <p class="vol-position-disciption-h2">Date: {{ opportunity.date }}</p>
             <p class="vol-position-disciption-h2">End: {{ opportunity.end }}</p>
             <p class="vol-position-disciption-h2">{{ opportunity.description }}</p>
+            {% if user.is_volunteer %}
+                {% if volunteer not in opportunity.volunteers.all %}
```

Comment:
Just a matter of preference, but could this be written as `if volunteer in opportunity.volunteers.all` instead and then flip the branches? It's a little easier to read positive conditionals than negative.